### ISSUE TITLE
Follow XDG Base Directory specification

### ISSUE THREAD

**Issue by sebdanielsson** [swiftbar/SwiftBar#232]
TLDR: Our home directories have turned into a miss of dotfiles and application directories. Most of my software now follow the XDG Base Directory spec by default or scans it before creating the config files in the home directory.

By following the spec SwiftBar would store config files at:
~/.config/swiftbar

The spec also recommends storing cache and other temporary files at:
~/.cache/swiftbar

The standard:
https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

Software that supports the XDG Base Directory standard:
https://wiki.archlinux.org/title/XDG_Base_Directory#Support
